### PR TITLE
Refine some metrics, remove name from TableReplicaInfo

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 

Fix https://github.com/pingcap/ticdc/issues/740

### What is changed and how it works?
- add metric to record processor resolved ts gap and checkpoint ts gap
- query table name from `SchemaStorage` when `addTable`

Screen Shot 2020-07-14 at 10 52 11

### Check List 

Tests 

 - Unit test
 - Integration test

### Release note

- No release note

### CODE REVIEW

File: cdc/processor.go [hunk 1]
```diff
@@ -363,7 +366,11 @@ func (p *processor) positionWorker(ctx context.Context) error {
 			}
 
 			p.position.ResolvedTs = minResolvedTs
-			resolvedTsGauge.Set(float64(oracle.ExtractPhysical(minResolvedTs)))
+			phyTs := oracle.ExtractPhysical(minResolvedTs)
+			resolvedTsGauge.Set(float64(phyTs))
+			// It is more accurate to get tso from PD, but in most cases we have
+			// deployed NTP service, a little bias is acceptable here.
+			metricResolvedTsLagGauge.Set(float64(oracle.GetPhysical(time.Now())-phyTs) / 1e3)
```
Comment (leoppro) [inline cdc/processor.go:373]: how about get a ts from PD?

Comment (amyangfei) [reply cdc/processor.go:373]: As the comment in L371-L372, it is a trade-off. Get `tso` from PD may fail, timeout or increase the load of PD (although this is not too frequent).

**Pull request merged**